tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -1531,35 +1531,35 @@ def create_full_val_real_complex128(val : 'float'):
     val_float = rand()*100
 
     f_int_int   = epyccel(create_full_val_int_int, language = language)
-    assert(     f_int_int(val_int)        ==      create_full_val_int_int(val_int))
+    assert      f_int_int(val_int)        ==      create_full_val_int_int(val_int)
```

Comment:
```suggestion
    assert f_int_int(val_int) == create_full_val_int_int(val_int)
```
Not here?

Reply:
Yes, here too!

Reply:
Done ad907a8422685d76810932d72f4959706fda2342